### PR TITLE
Add "High X Tolerance" and Night Raider traits to extra_mut_scen

### PR DESCRIPTION
**Pull request by KittyTac**
#### Summary
Bugfixes "Add "High X Tolerance" and Night Raider traits to extra_mut_scen"

#### Purpose of change
The mod didn't have those mutations.

#### Describe the solution
Add them to the list.

#### Describe alternatives you've considered
Making some kind of script to automatically add mutations to the mod; making it a test to force mutation contributors to add the traits they make to the mod so I don't have to act as its maintainer.

#### Testing
Tested and it works.

#### Additional context
Expect more like this eventually.

### CODE REVIEW

**Comment by KittyTac** [comment]
Any idea why this doesn't build?

**Comment by NetSysFire** [comment]
There is insane logspam and this was fixed yesterday. Try rebasing, it might fix it.

**Comment by KittyTac** [comment]
> There is insane logspam and this was fixed yesterday. Try rebasing, it might fix it.

Can you ELI5? I can't understand the guide Google brought up.

**Comment by NetSysFire** [comment]
When you make a new branch, you base it off master at a specific commit. Rebasing means you make it so that it will use the current latest commit in master instead, so all the new changes are pulled in, including the tests. Fortunately, this is just the press of a button in github (although you'd have to look up where it is) or simply `git rebase master` in a shell, without any other manual tweaking needed in both cases. It gets much more complicated when you have conflicts, but you do not so it will be a piece of cake and just requires finding and pressing the button.
